website/documentation/050-tutorials/content/howto/network-isolation/_index.md

```diff
@@ -100,7 +100,8 @@ kubectl apply -f ./network-policy.yaml -n=customer2
 
 after this `curl http://nginx.customer2` shouldn't work anymore if you are a service inside the namespace *customer1* and 
 vice versa
-  
+
+*Note*: This policy, once applied, will also disable all external traffic to these pods. I.e.: Internet, package management tools like apt-get, etc.
```

Comment:
`apt-get`s or any request from the containers to the outside world will work. If you create for example service of type LoadBalancer in this `customer1` ns that match the nginx deployment, and when you curl by external IP of the service then then network policy will deny the ingress traffic to the Pod.

```suggestion
*Note*: This policy, once applied, will also disable all external traffic to these pods. For example you can create a service of type `LoadBalancer` in namespace `customer1` that match the nginx pod. When you request the service by its <EXTERNAL_IP>:<PORT>, then the network policy will deny the ingress traffic from the service and the request will time out.
```

Reply:
Sorry, you're right. The ingress is locked.